styles/config/vocabularies/coco/accept.txt

```diff
@@ -2,3 +2,10 @@ Rivos
 Skopeo
 untrusted
 VM
+Nginx
+kubectl
+kubeconfig
+VNet
+subnet
```

Comment:
Shouldn't subnet be capitalized as an Azure API object, similar to VNet, Resource Group, etc...

Reply:
I checked various documentation, and it seems like subnet is referred more as a common noun than a proper noun. So, it can be referred to as subnet or Subnet depending on the capitalization rules.

Reply:
Like this: https://learn.microsoft.com/en-us/azure/virtual-network/concepts-and-best-practices